### PR TITLE
{bio}[foss/2020b] APBS v3.0.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)
requires #12587 (pdb2pqr)

### CODE REVIEW

**Comment by Micket** [comment]
Any reason why we don't compile this? We typically strive to avoid binary installs of software when we can avoid it.

**Comment by ThomasHoffmann77** [comment]
> Any reason why we don't compile this? We typically strive to avoid binary installs of software when we can avoid it.
 I assumed it would be better to at least have the binary install instead of nothing. I have been working on an installation from source. However, it is not ready, yet.

File: easybuild/easyconfigs/a/APBS/APBS-3.0.0_Linux.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# TH77EMBL update do 3.0.0, 2021/04
+name = 'APBS'
+version = '3.0.0'
+versionsuffix = '_Linux'
```
Comment (boegel) [inline easybuild/easyconfigs/a/APBS/APBS-3.0.0_Linux.eb:8]: Maybe tag this as `x86-64`? Linux is kind of a given...

But +1 on building this from source instead...

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**